releases/2.13.14.md

```diff
@@ -0,0 +1,54 @@
+# Scala 2.13.14 (DRAFT)
+
+The Scala team at Lightbend is pleased to announce Scala 2.13.14.
+
+The following changes are highlights of this release:
+
+### Align with Scala 3
+
+* Introduce `-Xsource-features`, for customizing the behavior of `-Xsource:3` and `-Xsource:3-cross` ([#10709](https://github.com/scala/scala/pull/10709))
```

Comment:
Add a link to https://docs.scala-lang.org/scala3/guides/migration/tooling-scala2-xsource3.html (with the updates in https://github.com/scala/docs.scala-lang/pull/2994)